tests/testthat/test-oc_process.R

```diff
@@ -323,3 +322,63 @@ test_that("oc_process handles various other arguments.", {
   expect_match(res3[[1]], "&add_request=1", fixed = TRUE)
   expect_match(res3[[2]], "&add_request=0", fixed = TRUE)
 })
+
+test_that("arguments that are NULL or NA don't show up in url.", {
+  res_null <- oc_process(
+    placename = "Hamburg",
+    return = "url_only",
+    key = NULL,
+    limit = NULL,
+    bounds = NULL,
+    proximity = NULL,
+    language = NULL,
+    countrycode = NULL,
+    min_confidence = NULL,
+    no_annotations = NULL,
+    no_dedupe = NULL,
+    no_record = NULL,
+    abbrv = NULL,
+    add_request = NULL
+  )
+  expect_match(res_null[[1]], "^((?!key=).)*$", perl = TRUE)
```

Comment:
:open_mouth: I am impressed by the regex :smile_cat:

Reply:
Me too! 😲